web/wp-content/themes/bumblebee/functions.php

```diff
@@ -156,7 +156,8 @@ function bumblebee_navigation_scripts() {
 	wp_deregister_script( 'jquery' );
 	wp_register_script( 'jquery', includes_url( '/js/jquery/jquery.js' ), false, '1.0.0', true );
 	wp_enqueue_script( 'slinky', get_stylesheet_directory_uri() . '/js/util/slinky.min.js', array( 'jquery' ), '4.1.0', true );
-	wp_enqueue_script( 'bumblebee-navigation', get_template_directory_uri() . '/js/navigation.js', array( 'slinky' ), '20190207', true );
+	wp_enqueue_script( 'bumblebee-navigation', get_template_directory_uri() . '/js/src/navigation.js', array( 'slinky' ), '20151215', true );
+	wp_enqueue_script( 'sticky-mobile-nav', get_stylesheet_directory_uri() . '/js/src/sticky-mobile-nav.js', array( 'jquery' ), '1.0.0', true );
```

Comment:
Is this file missing? https://pr-64-drdt.pantheonsite.io/wp-content/themes/bumblebee/js/src/sticky-mobile-nav.js

Reply:
Hmmm I don't know, it's in my local, and it says 'nothing to commit, working tree clean', how could that be?